lib/colorado_lottery.rb

```diff
@@ -0,0 +1,31 @@
+class ColoradoLottery
+  attr_reader :registered_contestants, :winners, :current_contestants
+
+  def initialize
+    @registered_contestants= {}
+    @winners = []
+    @current_contestants = {}
+  end
+
+  def interested_and_18?(contestant, game)
+    if contestant.game_interests.include?(game.name) && contestant.age > 18
```

Comment:
great! you don't need the if/else, you can just leave it as the boolean condition as it will return true/false on its own

```
  def interested_and_18?(contestant, game)
      contestant.age >= 18 && contestant.game_interests.include?(game.name)
  end
```